scripts/gee

```diff
@@ -3588,37 +3589,44 @@ EndOfPrTemplate
     #                 PR description.
   ) >"${BODYFILE}"
 
-  if [[ -n "${EDITOR}" ]]; then
-    "${EDITOR}" "${BODYFILE}"
-  else
-    declare -g RESP=""
-    _choose_one "How do you want to edit the PR description?? ([V]im, [e]macs, or [a]bort)  " \
-        "vVeEaA" "v"
-    case "${RESP}" in
-      [vV]*) vim "${BODYFILE}"; ;;
-      [eE]*) emacs "${BODYFILE}"; ;;
-      [aA]*) _fatal "Aborted."; ;;
-    esac
-  fi
+  DONE=0
+  while (( DONE == 0 )); do
+    if [[ -n "${EDITOR}" ]]; then
+      "${EDITOR}" "${BODYFILE}"
+    else
+      declare -g RESP=""
+      _choose_one "How do you want to edit the PR description?? ([V]im, [e]macs, or [a]bort)  " \
+          "vVeEaA" "v"
+      case "${RESP}" in
+        [vV]*) vim "${BODYFILE}"; ;;
+        [eE]*) emacs "${BODYFILE}"; ;;
+        [aA]*) _fatal "Aborted."; ;;
+      esac
+    fi
 
-  # Remove all comments from the description and squeeze multiple blank
-  # lines into single blank lines:
-  sed -i 's/^ *#.*//;/\S/,/^\s*$/!d' "${BODYFILE}"
+    # Remove all comments from the description and squeeze multiple blank
+    # lines into single blank lines:
+    sed 's/^ *#.*//;/\S/,/^\s*$/!d' "${BODYFILE}" > "${TRIMMED}"
 
-  if grep --quiet -E "^\s*ABORT" "${BODYFILE}"; then
-    _fatal "Aborting creation of new PR, as requested."
-    exit 1
-  fi
+    if grep --quiet -E "^\s*ABORT" "${TRIMMED}"; then
+      _fatal "Aborting creation of new PR, as requested."
+      exit 1
+    fi
 
-  # Check that the PR description looks valid
-  _check_pr_description "${BODYFILE}"
+    # Check that the PR description looks valid
+    if _check_pr_description "${TRIMMED}"; then
+      DONE=1
+    else
+      _info "Please fix your PR description."
```

Comment:
I assume it's fine that the user won't have time to see this info?

Reply:
I was worried about that, too (I had a sleep 1 here for a hot minute).  But: gee prompts for a choice of editor, so the user will have time to read the messages before the editor opens.  Oh -- unless the user has the EDITOR environment configured.  OK, I'll fix that.